client/store/reviews.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+import history from '../history'
+
+// ACTION TYPES
+const GET_ALL_REVIEWS = 'GET_ALL_REVIEWS'
+const POST_REVIEW = 'POST_REVIEW'
+
+//ACTION CREATORS
+const getReviews = reviews => ({type: GET_ALL_REVIEWS, reviews})
+const postReview = review => ({type: POST_REVIEW, review})
+
+//THUNK CREATORS
+export const getAllReviews = () => async dispatch => {
```

Comment:
It's worth thinking a bit about when we need to pull up reviews-- will we ever need to display *all* reviews? 

I can imagine us needing to display reviews by product, or perhaps by user... maybe best to start with displaying by product